Localize/update-locproject.ps1

```diff
@@ -0,0 +1,28 @@
+param ($SourcesDirectory, $LocProjectPath)

```

Comment:
1. Should this use a [shebang](https://stackoverflow.com/a/48218913) and be `+x` on Unix?
 2. If we *do* use a shebang, then this file needs to be Unix formatted.
 3. If this file needs to be Unix formatted, then we should update `.gitattributes` accordingly.

Reply:
I would lean towards no as I don't anticipate us needing to invoke this script much (if ever) outside of CI, and running the script with a `pwsh` prefix in those cases doesn't feel like too much overhead. I will make these changes if you would prefer though?